internal/ignition/boot-reporter/assisted-boot-reporter.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+export LOG_SEND_FREQUENCY_IN_MINUTES=5
+export SERVICE_TIMEOUT_MINUTES=60
+
+function log() {
+  echo "$(date '+%F %T') ${HOSTNAME} $2[$$]: level=$1 msg=\"$3\""
+}
+
+function log_info() {
+  log "info" "$1" "$2"
+}
+
+function log_error() {
+  log "error" "$1" "$2"
+}
+
+function init_variables() {
+  func_name=${FUNCNAME[0]}
```

Comment:
What is this FUNCNAME ? I don't see where we set it, sorry :(

Reply:
no need to set it ourselves: https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-FUNCNAME